src/main/features/claimant-response/routes/payment-plan.ts

```diff
@@ -26,9 +33,28 @@ class PaymentPlanPage extends AbstractPaymentPlanPage<DraftClaimantResponse> {
     return new DefaultModelAccessor('alternatePaymentMethod')
   }
 
-  buildTaskListUri (req: express.Request, res: express.Response): string {
-    const { externalId } = req.params
-    return Paths.taskListPage.evaluateUri({ externalId: externalId })
+  buildPostSubmissionUri (req: express.Request, res: express.Response): string {
+    const claim: Claim = res.locals.claim
+    const draft: DraftClaimantResponse = res.locals.draft.document
+    draft.acceptCourtOffer = undefined
+    draft.settlementAgreement = undefined
+    draft.formaliseRepaymentPlan = undefined
+
+    const courtOrder: CourtOrder = CourtOrderHelper.createCourtOrder(claim, draft)
+    const courtOrderAmount: number = courtOrder.calculateAmount()
+
+    logger.info(`Court redetermination amount: ${courtOrderAmount}, `
```

Comment:
As above, I'm not sure if this is something we want to be doing everything time this page is hit. Maybe we could create a support controller end point for querying a claim to get this information if the support team need it to trouble shoot a problem